index.js

```diff
@@ -22,7 +22,16 @@ class rpRetry {
                     logger.info(`Encountered error ${err.message} for ${options.method} request to ${options.uri}, retry count ${tryCount}`);
                     tryCount -= 1;
                     if (tryCount) {
-                        return fetchDataWithRetry(tryCount);
+                        let delay = options.delay || 100; // default delay between retries
```

Comment:
Consider documenting the units here: *default ms delay between retries*.

Reply:
Right, can do. Though as explained, I'm going to document this once we agree on the code/design itself.